src/test/java/gov/usgs/cida/pubs/webservice/mp/MpPublicationMvcServiceTest.java

```diff
@@ -97,7 +100,7 @@
 
 	@Before
 	public void setup() {
-		mvcService = new MpPublicationMvcService(pubBusService, busService);
+		mvcService = new MpPublicationMvcService(pubBusService, busService, sippService);
```

Comment:
I'd like to see some testing of the new end point in here. At the least, to show that the mock service gets called and to see that the posted parameters are validated.